pykCSD/KCSD2D.py

```diff
@@ -114,9 +114,9 @@ def set_parameters(self, params):
         nx = (self.xmax - self.xmin)/self.gdX + 1
         ny = (self.ymax - self.ymin)/self.gdY + 1
 
-        lin_x = np.linspace(self.xmin, self.xmax, nx)
-        lin_y = np.linspace(self.ymin, self.ymax, ny)
-        self.space_X, self.space_Y = np.meshgrid(lin_x, lin_y)
+        self.space_X, self.space_Y = np.mgrid[self.xmin:self.xmax:np.complex(0,nx),
+                                              self.ymin:self.ymax:np.complex(0,ny)]
```

Comment:
To avoid long lines:
```
        self.space_X, self.space_Y = np.mgrid[
            self.xmin:self.xmax:np.complex(0,nx),
            self.ymin:self.ymax:np.complex(0,ny)
        ]
```